Python/sysmodule.c

```diff
@@ -644,7 +645,8 @@ sys_breakpointhook(PyObject *self, PyObject *const *args, Py_ssize_t nargs, PyOb
 }
 
 PyDoc_STRVAR(breakpointhook_doc,
-"breakpointhook(*args, **kws)\n"
+"breakpointhook($module, /, *args, **kwargs)\n"
```

Comment:
`*args` is always positional, so the `/` comes after, no?

```suggestion
"breakpointhook($module, *args, /, **kwargs)\n"
```

Reply:
It is an invalid syntax. `/` always comes before `*`.